### ISSUE TITLE
Regarding  [Bag of Fishing Treasures] dropping Stormjewels!

### ISSUE THREAD

**Issue by Elefant0n** [OmegaWoWeu/Decades#187]
Description:  [Bag of Fishing Treasures] is supposed to drop Stormjewels, What i'm wondering is, 
Is it added to the loot table ? I haven't seen any of them so far and we're quite a bit into 3.1.

How to reproduce bug:
Not a bug.

How it should work:
Drop Stormjewel.

Screenshots: None required.

Relevant AddOns used: None.

Source: (https://www.wowhead.com/item=45862/bold-stormjewel)
(Added in patch 3.1.0.9658)

**Comment by JonasD92** [comment]
They have a drop chance of <1%. How many bags did you open?

**Comment by Elefant0n** [comment]
I've opened 25 so far, But i don't know if the bags we're switched to the right ID after the Ulduar release so might have just been 6.

The thing i'm trying to bring up is, IS it added to the loot table ? Since i'm in contact with most of the guilds on the server and i have not SEEN one equipped or SEEN one on the auction house.
This is why i drew the conclusion of it missing from the loot table.

These usually end up on the auction house straight away since most players value the 2-3k instead of an 4~ stat increase.

**Comment by JonasD92** [comment]
These bags were introduced in the same patch as the stormjewels. Were these bags available on the server before the ulduar content? Because if they just added them in, they didn't have any reason to remove the epic gems from the loot table, so there is no reason to think that they are removed.

If they were already available on the server before Ulduar, then I just don't get it why they added the 3.1 quests to 3.0 content but manually took out the epic gems. I'm not asking this from you of course, it's poetic.

We really need a server staff to answer this then.

**Comment by Elefant0n** [comment]
I do appreciate you responding but please read up before you start asking dumb questions. >.<

These bags does exist before 3.1 but does not have the Stormjewels in the Loot table. (And this is how it was in Retail back in the days aswell)
Seeing how these bags keep the same name but change Drop table with the release of Ulduar, My first thought was that the Dev's prob forgot to change it....

Yes... Yes we do.

**Comment by JonasD92** [comment]
The bag of fishing treasure which yourself linked is added in the same patch as the stormjewels.
https://www.wowhead.com/item=46007/bag-of-fishing-treasures
I clicked on your link and clicked on the bag in the 'Contained in' section. It was added 3.1.0.9658. Same as the stormjewels. 

Also the quests that give these bags are introduced in 3.1:
https://wowwiki.fandom.com/wiki/Marcia_Chase
Patch 3.1.0 (14-Apr-2009): Daily quests added.

The other bags of fishing treasure(lvl 70 ones) never had the epic gems.

Which statement of mine was stupid?
